test/unit/presenters/search_results_presenter_test.rb

```diff
@@ -138,14 +138,7 @@ class SearchResultsPresenterTest < ActiveSupport::TestCase
         start: 100,
       })
 
-      assert_equal 50, params.count
-      presenter = SearchResultsPresenter.new(response, params)
```

Comment:
I'm not sure we want to drop these lines of testing. Are they duplicated somewhere else?